src/Simplex/Chat/Protocol.hs

```diff
@@ -316,7 +318,8 @@ instance ToJSON LinkPreview where
 data MsgContent
   = MCText Text
   | MCLink {text :: Text, preview :: LinkPreview}
-  | MCImage {text :: Text, image :: ImageData}
+  | MCImage {text :: Text, image :: ImageData, fileDescr :: Maybe Text}
+  | MCVideo {text :: Text, poster :: ImageData}
```

Comment:
why it doesn't need fileDescr same as image?

Reply:
This fileDescr is specifically for low-resolution image in case a high-resolution image is passed as an attachment. I thought about supporting multiple attachments, and it seems like it is too complex in this case - we would have to support multiple files per message... Although the schema seems to allow multiple files per message, unless I'm mistaken, so maybe we should do it this way. But this is a case when we have a variant of the file - a low resolution option to complement a higher resolution option. Possibly we can allow the same for vides. This is different from multiple attachments.

Reply:
although I guess we can have lower resolution video as well, but I don't think it needs to happen in v1

Reply:
and maybe never